content/blog/2016/2016-05-04-the-state-of-JAM

```diff
@@ -0,0 +1,44 @@
+---
+layout: post
+title: "The State of Jenkins Area Meetup (JAM)"
+tags:
+- meetup
+- JAM
+- event
+author: alyssat
+---
+
+NOTE: _This is a guest post by link:https://github.com/alyssat[Alyssa Tong].
```

Comment:
I agree with @rtyler. Would be better if you use your own name, Hannah. Writing content for jenkins.io is also a contribution to the Jenkins project, and we would like to see as many authors as possible.